test/scenarios/el94-src@low-latency.sh

```diff
@@ -15,5 +15,14 @@ scenario_remove_vms() {
 }
 
 scenario_run_tests() {
-    run_tests host1 suites/tuned/
+    # Should not be ran immediately after creating VM because of
+    # microshift-tuned rebooting the node to activate the profile.
```

Comment:
How do we time this after creating the VM, then?

Reply:
Should we add some kind of a loop checking that the profile is active?

Reply:
Added a check to see if we have `>= 2` boots. `>` so we can run the script repeatedly in local env.